tests/tools.bats

```diff
@@ -19,79 +23,103 @@ teardown () {
 }
 
 @test "Testing AWS docker container: should be successful" {
+  # arguments initialization (inputs)
```

Comment:
I would keep the capitalization consistent. For instance first letter capitalized, as in a regular human sentence.